compile-test/crmTests/logs/conversion80000-92000.log

```diff
@@ -0,0 +1,9 @@
+# Conversion log for project 'crmTests' from version 80000 to 92000
```

Comment:
do not commit this *.log files.
thanks. and have a nice day.

Reply:
Already removed it with next commit. I've also migrated the base, customer, solution projects (those for the deploy scenario)